alpine-consul/root/etc/services.d/consul-template/run

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/with-contenv sh
```

Comment:
@matthewvalimaki, I was reading the `consul-template` documentation a little more (I haven't actually used it before). Could we make use of the `retry` option? Would that cause `consul-template` not to error? Perhaps then we could remove the Consul nodes watch?